### ISSUE TITLE
Getting Low mIOU on pretrained models.

### ISSUE THREAD

**Issue by AndreisBruno** [hszhao/semseg#15]
When performing evaluation on the Cityscapes validation set using the provided pspnet_resnet101 pretrained model, I get a much lower result than stated. Anything im missing?

`[2019-09-20 22:33:20,678 INFO test.py line 249 24510] Eval result: mIoU/mAcc/allAcc 0.0039/0.0165/0.0077.
[2019-09-20 22:33:20,678 INFO test.py line 251 24510] Class_0 result: iou/accuracy 0.0000/0.0001, name: road.
[2019-09-20 22:33:20,678 INFO test.py line 251 24510] Class_1 result: iou/accuracy 0.0051/0.0112, name: sidewalk.
[2019-09-20 22:33:20,678 INFO test.py line 251 24510] Class_2 result: iou/accuracy 0.0149/0.1796, name: building.
[2019-09-20 22:33:20,678 INFO test.py line 251 24510] Class_3 result: iou/accuracy 0.0032/0.0045, name: wall.
[2019-09-20 22:33:20,678 INFO test.py line 251 24510] Class_4 result: iou/accuracy 0.0350/0.0555, name: fence.
[2019-09-20 22:33:20,678 INFO test.py line 251 24510] Class_5 result: iou/accuracy 0.0097/0.0398, name: pole.
[2019-09-20 22:33:20,678 INFO test.py line 251 24510] Class_6 result: iou/accuracy 0.0000/0.0000, name: traffic light.
[2019-09-20 22:33:20,678 INFO test.py line 251 24510] Class_7 result: iou/accuracy 0.0001/0.0001, name: traffic sign.
[2019-09-20 22:33:20,678 INFO test.py line 251 24510] Class_8 result: iou/accuracy 0.0008/0.0035, name: vegetation.
[2019-09-20 22:33:20,678 INFO test.py line 251 24510] Class_9 result: iou/accuracy 0.0030/0.0087, name: terrain.
[2019-09-20 22:33:20,678 INFO test.py line 251 24510] Class_10 result: iou/accuracy 0.0000/0.0000, name: sky.
[2019-09-20 22:33:20,678 INFO test.py line 251 24510] Class_11 result: iou/accuracy 0.0015/0.0016, name: person.
[2019-09-20 22:33:20,678 INFO test.py line 251 24510] Class_12 result: iou/accuracy 0.0004/0.0005, name: rider.
[2019-09-20 22:33:20,678 INFO test.py line 251 24510] Class_13 result: iou/accuracy 0.0005/0.0046, name: car.
[2019-09-20 22:33:20,678 INFO test.py line 251 24510] Class_14 result: iou/accuracy 0.0000/0.0000, name: truck.
[2019-09-20 22:33:20,678 INFO test.py line 251 24510] Class_15 result: iou/accuracy 0.0001/0.0013, name: bus.
[2019-09-20 22:33:20,678 INFO test.py line 251 24510] Class_16 result: iou/accuracy 0.0000/0.0000, name: train.
[2019-09-20 22:33:20,678 INFO test.py line 251 24510] Class_17 result: iou/accuracy 0.0005/0.0005, name: motorcycle.
[2019-09-20 22:33:20,678 INFO test.py line 251 24510] Class_18 result: iou/accuracy 0.0003/0.0013, name: bicycle.
`

**Comment by hszhao** [comment]
Hi, are you using the right gt labels? You need to use the train id instead of the label id. https://github.com/mcordts/cityscapesScripts/blob/master/cityscapesscripts/helpers/labels.py

**Comment by fido20160817** [comment]
> Hi, are you using the right gt labels? You need to use the train id instead of the label id. https://github.com/mcordts/cityscapesScripts/blob/master/cityscapesscripts/helpers/labels.py

how to deal with ade20k: Eval result: mIoU/mAcc/allAcc 0.0008/0.0021/0.0193. by running test.py?
